miden/src/cli/prove.rs

```diff
@@ -0,0 +1,67 @@
+use super::data::{InputFile, ProofFile, ScriptFile};
+use air::ProofOptions;
+use log::debug;
+use std::path::PathBuf;
+use std::time::Instant;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "Prove", about = "Prove a miden program")]
+pub struct ProveCmd {
+    /// Path to .masm script file
+    #[structopt(short = "s", long = "script-file", parse(from_os_str))]
```

Comment:
Both `script-file` and `security` have `s` for short command - wouldn't this conflict? Maybe use `a` for short and `assembly-file` for long?

Reply:
good catch will change